app/models/concerns/foreman_puppet/extensions/hostgroup.rb

```diff
@@ -17,6 +17,7 @@ class << self
         # will need through relation to work properly
         scoped_search relation: :environment, on: :name, complete_value: true, rename: :environment, only_explicit: true
         scoped_search relation: :puppetclasses, on: :name, complete_value: true, rename: :class, only_explicit: true, operators: ['= ', '~ ']
```

Comment:
Do we want to keep the `class`  search parameter or it's time to remove it?

Reply:
Personally, I would remove it but then we should also remove it for hosts.